src/styles/cards.scss

```diff
@@ -22,6 +22,7 @@
 
   &--logo {
     width: 75px;
+    margin-top: 3px;
```

Comment:
Giving the card logos button has a fixed height from the top
![image](https://user-images.githubusercontent.com/63985172/135029699-20f37893-b8e8-41aa-902f-090cdb877d06.png)